lighthouse-plugin-publisher-ads/utils/resource-classification.js

```diff
@@ -37,12 +37,65 @@ function isGoogleAds(url) {
 }
 
 /**
- * Checks if the url is for pubads implementation tag.
+ * Checks if the url is loading an AdSense script.
  * @param {URL|string} url
  * @return {boolean}
  */
-function isImplTag(url) {
-  return /(^\/gpt\/pubads_impl_\d+.js)/.test(toURL(url).pathname);
+function isAdSenseTag(url) {
+  const {host, pathname} = toURL(url);
+  const matchesHost = [
+    'pagead2.googlesyndication.com'].includes(host);
```

Comment:
Nit: just check for string equality here and in `isAdSenseImplTag`?

Reply:
Done